packages/cli/src/util/projects/update-project.ts

```diff
@@ -1,5 +1,5 @@
 import Client from '../client';
-import { ProjectSettings } from '../../types';
+import { JSONObject, ProjectSettings } from '../../types';
```

Comment:
Do we ever use `import type ...` if we're only importing types?

Reply:
Ideally yes, but we don't have a lint rule set up for that yet (we should use `@vercel/style-guide` 😬).

Reply:
Just imported type!